odoo/tests/common.py

```diff
@@ -78,6 +78,10 @@
 _logger = logging.getLogger(__name__)
 
 
+DEFAULT_TESTS_CONTEXT = {
+    'no_reset_password': True,
+}
```

Comment:
Shouldn't this be a class method so that each test class can define their own context keys if needed?
Or remove this key if we want to test the passwords